src/translations/en.json

```diff
@@ -1067,7 +1067,9 @@
             "warning": "WARNING",
             "info": "INFO",
             "debug": "DEBUG"
-          }
+          },
+          "custom_component": "custom component",
+          "error_from_custom_component": "This error originated from a custom component."
```

Comment:
```suggestion
          "error_from_custom_integration": "This error originated from a custom integration."
```

Reply:
I think we should add a bit on what it means? E.g., if-this-is-an-issue-report-it-with-the-author-of-this-custom-integration-ish.